src/Stryker.Core/Stryker.Core.UnitTest/Mutants/CsharpMutantOrchestratorTests.cs

```diff
@@ -815,6 +826,21 @@ public void ShouldNotMutateIfDisabledByComment()
             ShouldMutateSourceInClassToExpected(source, expected);
         }
 
+        [Fact]
+        public void ShouldNotMutateIfDisabledByCommentOnStaticFields()
+        {
+            var source = @"
+    // Stryker disable all
+    static string x = ""test"";";
+
+            var expected = @"
+    // Stryker disable all
+    static string x = StrykerNamespace.MutantContext.TrackValue(()=>""test"");";
+            ShouldMutateSourceInClassToExpected(source, expected);
+
+
```

Comment:
```suggestion

            ShouldMutateSourceInClassToExpected(source, expected);
```

Reply:
lol, I spotted the same problem at the same time :-)

Reply:
It looked like that yes :)